mava/logger.py

```diff
@@ -36,10 +36,22 @@ def __call__(
         ...
 
 
-def get_logger_tools(logger: Logger) -> LogFn:  # noqa: CCR001
+class SebulbaLogFn(Protocol):
```

Comment:
I really don't think this is a good idea, can we please use the existing log_fn otherwise there will be a lot of code duplication

Reply:
I tired to address this issue in #977